client/src/actions/ActionTypes.js

```diff
@@ -0,0 +1,10 @@
+import keyMirror from 'keymirror';
+
+export default keyMirror({
+  AUTH_ERROR: null,
+  CHECK_IN_SUCCESS: null,
+  SIGN_OUT: null,
+
```

Comment:
Why this space, also a bonus point would be making your actions here alphabetical. You don't need to do it though